$OpenBSD: patch-src_CMakeLists_txt,v 1.2 2016/11/17 09:51:41 fcambus Exp $

Prevent libzip's zip.h header to be picked up instead of src/includes/zip.h

--- src/CMakeLists.txt.orig	Thu Nov  3 22:25:43 2016
+++ src/CMakeLists.txt	Fri Nov  4 20:38:26 2016
@@ -43,8 +43,8 @@ else()
 	set(CPUDIR uae-cpu)
 endif(ENABLE_WINUAE_CPU)
 
-include_directories(${CMAKE_BINARY_DIR} ${SDL_INCLUDE_DIR} ${CPUDIR}
-		    includes debug falcon)
+include_directories(includes debug falcon
+		    ${CMAKE_BINARY_DIR} ${SDL_INCLUDE_DIR} ${CPUDIR})
 
 if(ZLIB_FOUND)
 	include_directories(${ZLIB_INCLUDE_DIR})
